src/utils/motebehovUtils.ts

```diff
@@ -19,38 +19,25 @@ export const sorterMotebehovDataEtterDato = (
     : -1;
 };
 
+// Møtebehov av typen SVAR_BEHOV med harMotebehov = false genererer ikke oppgaver og skal ikke behandles
 export const motebehovUbehandlet = (
   motebehovListe: MotebehovVeilederDTO[]
 ): MotebehovVeilederDTO[] => {
   return motebehovListe.filter(
     (motebehov) =>
       motebehov.motebehovSvar &&
       motebehov.motebehovSvar.harMotebehov &&
-      !motebehov.behandletTidspunkt
+      !motebehov.behandletTidspunkt &&
+      !motebehov.behandletVeilederIdent
   );
 };
 
-export const getUbehandletSvarOgMeldtBehov = (
```

Comment:
Ingrid og jeg lagde denne sist, men det blir litt feil å bare se på `behandletTidspunkt` og `behandletVeilederIdent` ettersom disse kan være `null` for møtebehovene der aktøren sier de ikke ønsker møte (`harMotebehov = false`), og dermed komme gjennom filteret her.